src/System.Slices/src/System/Span.cs

```diff
@@ -335,7 +320,7 @@ public override bool Equals(object obj)
         /// gain: performance: no boundaries check (single operation) 
         /// </summary>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        private T GetItemWithoutBoundariesCheck(int index)
+        internal T GetItemWithoutBoundariesCheck(int index)
```

Comment:
Historically we've used the word "Dangerous" for these sorts of methods.  However, I don't think we should expose something like this as it makes it very easy to break type safety if `PtrUtils.Get<T>` doesn't do bounds checking.